docs/docs/build/getting-started.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Your First Rill Project"
+description: "Get started with Rill by understanding what happens when you create a project"
+sidebar_label: "Getting Started"
+sidebar_position: 1
+---
+
+# Your First Rill Project
+
+When you create a new Rill project, you're setting up a complete data pipeline that transforms raw data into interactive dashboards. This guide explains what happens during project initialization and introduces you to the key files you'll work with.
+
+## What Happens When You Create a Rill Project?
+
+A Rill project consists of several data assets that work together to create a dashboard. The data pipeline begins with [connecting](/connect) to your data sources, transforms raw data through [models](/build/models), defines metrics and dimensions in [metrics views](/build/metrics-view), and results in interactive [dashboards](/build/dashboards) for data analysis.
+
+When you create a new Rill project, the following files are automatically generated:
+
+- **`rill.yaml`** - Central configuration hub for your entire project
+- **`connectors/<connector>.yaml`** - Connector configuration files for default OLAP engine (e.g., `duckdb.yaml`, `clickhouse.yaml`)
+- **`.gitignore`** - Git ignore rules for the project
+
+## Project Files Overview
```

Comment:
This title feels a bit misleading– it somewhat implies that the below files are the only kinds of files in a project. Rather, the below files are merely the _initial_ project files.

It could help to clarify that projects can and will contain many more types of files.